packages/module/src/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import { CloseIcon } from '@patternfly/react-icons';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
+  '& float-right': {
+    float: 'right'
+  },
+  '&  no-padding': {
+    padding: 0
+  }
+});
+
+export interface CloseButtonProps {
```

Comment:
```suggestion
export interface CloseButtonProps extends ButtonProps {
```
we should extend the ButtonProps interface to support all Button props